db/view/view.cc

```diff
@@ -2469,6 +2469,11 @@ static future<> flush_base(lw_shared_ptr<replica::column_family> base, abort_sou
 }
 
 void view_builder::on_create_view(const sstring& ks_name, const sstring& view_name) {
+    if (_db.find_keyspace(ks_name).uses_tablets()) {
```

Comment:
In the final version, we should probably add this in the same commit where we switch to the view building coordinator for views on tablets, otherwise we might get some errors during bisects where a view with tablets can't get built at all